technic/tools/init.lua

```diff
@@ -1,4 +1,5 @@
 local path = technic.modpath.."/tools"
+local mesecons_materials = minetest.get_modpath("mesecons_materials")
```

Comment:
With tools/init.lua reset was referring to this and related check (crafting guides should be able to handle missing items):
```suggestion
```